app/tests/spec/lib/router.js

```diff
@@ -88,6 +90,69 @@ function (chai, sinon, Backbone, Router, SignInView, SignUpView, ReadyView,
       $('#container').empty();
     });
 
+    describe('routes', function () {
+      beforeEach(function () {
+        sinon.stub(window.history, 'pushState', function () {
+        });
+        Backbone.Router.prototype.navigate = origNavigate;
```

Comment:
Should we just clean the tests up here and use a sinon.sandbox on the Backbone.router.prototype for `navigate` & restore the sandbox before these tests? I probably did that origNavigate stuff before I learned how to use sinon stubs & sandboxes.